src/main/java/org/kuali/kfs/pdp/batch/service/impl/Iso20022FormatExtractor.java

```diff
@@ -375,6 +375,7 @@ public void extractChecks(
                     message.setCstmrCdtTrfInitn(customerCreditTransferInitiation);
 
                     final String filename = determineFilename(directoryName, extractTypeContext);
+                    LOG.info("extractChecks writing file " + filename);
```

Comment:
There's an equivalent spot in the `extractAchs()` method where another logging statement should be inserted. Alternatively, you could move the custom Check/ACH filename logging to the `writeMessageToFile()` method.